chaos-monkey-docs/src/main/asciidoc/changes.adoc

```diff
@@ -4,14 +4,13 @@
 Built with Spring Boot {spring-boot-version}
 
 === Bug Fixes
-- https://github.com/codecentric/chaos-monkey-spring-boot/pull/251[#251] Fix missing AssaultPropertiesUpdate validation.
```

Comment:
These line should stay here

Reply:
> Always create a separate branch for your contributions. So you can keep your main branch up to date with upstream/original repository (either through git tools or github ui, see screenshot)

Yeah, I realized that after I did the pull request already. I initially thought it would be fine since I needed to do this to run the project on my system; though realized I couldn't update the master branch with new commits until this pull request closed. 😅